client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -6,6 +6,7 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.ObjectReader;
 import com.fasterxml.jackson.databind.type.TypeFactory;
+import com.netflix.servo.util.Strings;
```

Comment:
I don't think we have an explicit dependency on servo... And we should eventually get rid of servo. If apache is not on the classpath then perhaps just write the code yourself?

Reply:
The import is still here?

Reply:
Sorry! Will remove